recipes/dmff/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dmff" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/DMFF/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d990516e2dc9ba45d87a8a93fc6fc12bc542adf555bba3e706a41e4db1655d71
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - numpy >=1.18
+    - jax
+    - jaxlib
+    - jax-md >=0.1.28
+    - mdtraj
+    - optax
+    - pymbar
+    - openmm >=7.6.0
+    - rdkit
+    - freud
+
+test:
+  imports:
+    - dmff
+
+about:
+  home: https://github.com/deepmodeling/DMFF
+  license: LGPL-3.0-only
+  license_family: LGPL
+  license_file: LICENSE
+  summary: DMFF (Differentiable Molecular Force Field) is a Jax-based python package that provides a full differentiable implementation of molecular force field models.
+  doc_url: https://github.com/deepmodeling/DMFF
+  dev_url: https://github.com/deepmodeling/DMFF
+
+extra:
+  recipe-maintainers:
+    - njzjz
+    - WangXinyan940
```

Comment:
Hi @WangXinyan940, please post a comment confirming you are willing to be listed here.